src/data/faq.json

```
@@ -844,7 +844,9 @@
                             "<b>5. Can I warn for others regardless of which QR code I have scanned?</b>",
                             "Organizers can warn for others regardless of which QR code they have scanned with the CWA. For more information on warning for others, see the following links:<ul><li><a href='#warn_for_others' target='_blank'>FAQ: How can I warn for others?</a></li><li><a href='../blog/2021-09-08-cwa-version-2-9/' target='_blank'>Blog: Corona-Warn-App version 2.9: Organizers can use the check-in function to warn guests on behalf of the health department</a></li></ul>",
                             "<b>6. Can the Corona-Warn-App read and process the QR codes from luca?</b>",
-                            "Yes. The Corona-Warn-App can scan QR codes issued for event registration using the luca app and perform its own check-in process from the CWA.",
+                            "Yes. The Corona-Warn-App can scan printed out QR codes issued by business owners for event registration with luca and perform its own CWA check-in process.",
+                            "The QR codes shown inside the luca app on smartphones (for private meetings and for check-in through a scanner owned by the location) have different use-cases than and are therefore not supported by the CWA.",

```

Two possible suggestions:

Either drop the "than" completely (analogous to the German translations):
```suggestion
                            "The QR codes shown inside the luca app on smartphones (for private meetings and for check-in through a scanner owned by the location) have different use-cases than checking in to events and are therefore not supported by the CWA.",
```
Or add to what it's different:
```suggestion
                            "The QR codes shown inside the luca app on smartphones (for private meetings and for check-in through a scanner owned by the location) have different use-cases and are therefore not supported by the CWA.",
```

---

The second, simpler, version is better i.e. just delete "than".